retrofit2/src/test/java/com/linecorp/armeria/client/http/retrofit2/ArmeriaCallFactoryTest.java

```diff
@@ -309,4 +287,15 @@ public void onFailure(Call<Pojo> call, Throwable t) {
         pojoCall.cancel();
         assertThat(failCount.intValue()).isZero();
     }
+
+    @Test
+    public void respectsHttpClientUri() throws Exception {
```

Comment:
Right, need to fix. Actually, when I try to use EndpointGroup, it does not work.
```
java.lang.IllegalArgumentException: Illegal URL: http://group:group/
	at retrofit2.Retrofit$Builder.baseUrl(Retrofit.java:437)
	at com.linecorp.armeria.client.http.retrofit2.ArmeriaRetrofit.builder(ArmeriaRetrofit.java:53)
```
I need to add some changes to ArmeriaCallSubscriber to update okhttp's url